index.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="initial-scale=1, width=device-width" />
+    <meta charset="utf-8"/>
```

Comment:
I'm a little confused over these changes with the spaces inside the meta and link tags in Line 4, 5, 7, and 8? I think we could leave them alone.